src/components/generic/button/Button.scss

```diff
@@ -1,54 +1,72 @@
-.h5p-hub-publish,
-.h5p-hub-registration {
-  button {
-    font-family: "Open Sans", sans-serif;
-    font-size: 0.75em;
-    background: transparent;
-    padding: 0.75em 2em;
-    color: #757575;
-    border-radius: 3em;
-    border: 1px solid #d6d6d6;
-    cursor: pointer;
-    font-weight: bold;
-
-    &:before {
-      font-weight: normal;
-    }
+.btn {
```

Comment:
Can you please explain why you have changed the selector here? This seems like it is very unspecific and will potentially affect other buttons on the page which we don't intend to change.

The previous selector ensured the rules within would not affect anything outside the scope of ".h5p-hub-publish" or ".h5p-hub-registration".

Reply:
I'm okay with it :)